OxfordBodleian/Juel-Jensen/BDLaethf25.xml

```diff
@@ -809,18 +809,19 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                   The boards are flush the bookblock and have external shallow bevels. The grain runs vertically.
                                 </decoNote>
                                 <decoNote xml:id="b3" type="Cover" color="reddish-brown">
-                                  The boards and the spine are partially covered with plain leather.
+                                  The boards and the spine are partially covered with plain reddish-brown leather.
```

Comment:
Does "plain" mean "undecorated"? To me, it's not the clearest term

Reply:
I think plain is fine!

Reply:
And how do you understand it?